include/drivers/serial.h

```diff
@@ -98,10 +99,17 @@ union interrupt_enable_register {
 };
 typedef union interrupt_enable_register ier_t;
 
+#define COM1_IRQ0_OFFSET (PIC_IRQ0_OFFSET + 4)
```

Comment:
nit: it might make sense to keep track of allocated irqs in a variable and/or dedicated interface implemented in PIC

Reply:
Moved the logic to enable an irq line; I wouldn't keep a variable as it's easy to get the allocated irqs (and it's already stored somewhere), but I can add that too